vm/proxyapp/proxyappclient.go

```diff
@@ -107,6 +107,12 @@ func (p *pool) init(params *proxyAppParams, cfg *Config) error {
 		}
 	}
 
+	err = p.proxy.Init()
```

Comment:
Why is this necessary provided we have CreatePool call below?

Reply:
I needed some global initializer and forgot about the package level "init()" function.

Reply:
Also main function will work and we also call CreatePool when start the app.

Reply:
I wanted to initialize plugin logger. "init()" looks better for that than "CreatePool()".
main() - thanks... agree 👍